client/src/components/PropertiesList.tsx

```diff
@@ -62,7 +69,48 @@ const useValue = () => {
 
 export const FilterContext = React.createContext({} as ReturnType<typeof useValue>);
 
-const FilterContextProvider: React.FC<{}> = (props) => {
+export const filterAddrs = (addrs: AddressRecord[], filterSelections: FilterSelections) => {
+  const {
+    rsunitslatest: filterRsunitslatest,
+    ownernames: filterOwnernames,
+    unitsres: filterUnitsres,
+    zip: filterZip,
+  } = filterSelections;
+
+  return addrs.filter((addr) => {
+    let keepAddr = true;
```

Comment:
weirdly, i don't mind local variables with shorthand names

Reply:
makes sense, I saw a lot of use of `addrs` as prop names and such, but I agree for function/component names the full word makes sense.